fedscale/core/execution/android/README.md

```diff
@@ -0,0 +1,45 @@
+# Android Sample App
+
+This directory contains minimum files modified from [MNN Android Demo](https://github.com/alibaba/MNN/tree/master/project/android/demo). The training and testing will be conducted by MNN C++ backend, while the task execution and communication with server will be managed by Java. The sample has been tested upon image classification with a simple linear model and a small subset of [ImageNet-MINI](https://www.kaggle.com/datasets/ifigotin/imagenetmini-1000). This documentation contains a step-by-step tutorial on how to download, build and config this app on your own device, and modify this app for your own implementation and deployment.
+
+## Download and build sample android app
+
+1. Download and unzip [sample dataset (TrainTest.zip)](https://drive.google.com/uc?export=download&id=1sLKIx6kuVZ0jVGgBLx_AGTM0YM5rhct_) to `assets/` directory. Remove `TrainTest.zip` after unzip to save space on your mobile device. After unzip, you should see 3 files and 2 directories under `assets/`:
+   1. `TrainSet`: Training set directory, contains 316 images.
+   2. `TestSet`: Testing set directory, contains 34 images.
+   3. `conf.json`: Configuration file for mobile app.
+   4. `train_labels.txt`: Training label file with format `<filename> <label>`, where `<filename>` is the path after `TrainSet/`.
+   5. `test_labels.txt`: Testing label file with the same format as `train_labels.txt`.
+2. Run `sh install.sh`. If you are on Windows, please run this command in WSL. This step will clone MNN, replace [MNN Android Demo](https://github.com/alibaba/MNN/tree/master/project/android/demo) with FedScale Android Executor under `MNN/project/android/demo`. **Note**: you may want to move MNN directory to another directory, otherwise the full file path might be too long for cmake to build.
+3. Install [Android Studio](https://developer.android.com/studio) and open project `MNN/project/android/demo`. Download necessary SDKs, NDKs and CMake when prompted. My version:
+    - SDK: API 28
+    - NDK: 20.0.5594570
+    - Android Gradle Plugin Version: 3.5.3
+    - Gradle Version: 5.4.1
+    - Source Compatibility: Java 8
+    - Target Compatibility: Java 8
+4. Make project. Android Studio will compile and build the app for you.
+
+## Test this app with default setting
+
+1. ssh to your own server and run
+```
+git clone https://github.com/SymbioticLab/FedScale.git
+cd FedScale
+source install.sh
+pip install -e .
+cd fedscale/core/aggregation/android
+python3 android_aggregator.py --experiment_mode=mobile --num_participants=1 --model=linear
+```
+2. Change aggregator IP address inside `conf.json` and click `Run` inside Android Studio.
```

Comment:
explicitly mention the path of `conf.json`

Reply:
Sure. It is assets/conf.json, I will update the pull request later.